### PR TITLE
Add secrets command to CLI 

### PR DESCRIPTION
**Pull request by anna-cross**
## Description of change

We want to be able to add, list, describe , remove and update secrets from CLI. 

Fixes https://github.com/meroxa/mdpx/issues/656

## Type of change

- [ ]  New feature
- [ ]  Bug fix
- [x]  Refactor
- [ ]  Documentation

## How was this tested?

- [x]  Unit Tests
- [x]  Tested in staging
- [ ]  Tested in minikube

## Demo

(In slack)

### CODE REVIEW

**Review by nassor — APPROVED** [review]
Nice little refactorings as well :)

File: utils/display/display.go [hunk 1]
```diff
@@ -71,11 +73,13 @@ func PrintList(input interface{}, details Details) string {
 			fmt.Printf("err: %v\n", err)
 			return ""
 		}
-
 		row := []*simpletable.Cell{}
-		for _, field := range details {
+
+		for _, o := range hh {
+			// fmt.Printf("Field: %s\n", o)
+			// fmt.Printf("Value: %v\n", amorphous[o])
```
Comment (samirketema) [inline utils/display/display.go:80]: perhaps leftover from debugging?

**Review by samirketema — APPROVED** [review]
LGTM, one comment about commented code we can remove

File: utils/display/display.go (hunk 1 above)
Comment (anna-cross) [reply utils/display/display.go]: Yup! removing it!

**Pull request merged**